src/domain/home/form/partial/BirthdateFormField.tsx

```diff
@@ -12,7 +12,7 @@ const BirthdateFormField: FunctionComponent<FieldArrayRenderProps> = ({
   form: { errors, touched },
 }) => {
   const fieldTouched = getIn(touched, 'child.birthdate.day');
-  const error = errors['childBirthdate'];
+  const error = getIn(errors, 'childBirthdate');
```

Comment:
this could be done with plain typescript 3.7:
```
const error = errors?.childBirthdate
```

Reply:
that's nice, but our ts version is lower than 3.7 
Will take this in use after upgrade, would be nice to remove lot of hacky nullcheck 👍